shopinvader_product/schemas/category.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from __future__ import annotations
```

Comment:
PQ cet import?

Reply:
Because the model is self-referencing and we cannot refer to the model using a string because of the `| None`.

Python does not understand what we are doing and returns a `TypeError: unsupported operand type(s) for |: 'str' and 'NoneType'`